ui/.env.development

```diff
@@ -14,4 +14,13 @@ REACT_APP_USER_DOCS_URL=[{"href":"https://github.com/gojek/turing","label":"Turi
 # Link to the .proto file used by the Kafka Result Logger when using Protobuf serialization
 REACT_APP_RESULT_LOG_PROTO_URL=
 
+# Default Experiment Engine Remote UI settings
+REACT_APP_DEFAULT_EXPERIMENT_ENGINE_REMOTE_APPNAME=
+REACT_APP_DEFAULT_EXPERIMENT_ENGINE_REMOTE_URL=
```

Comment:
Since these two variables needs to be defined simultaneously (either both or none), they can be combined into a single env variable defined as JSON object, similar to `REACT_APP_USER_DOCS_URL`.